<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>PlainText</string>
       </attribute>
       <widget class="QHexView" name="plainHexView">
        <property name="geometry">
         <rect>
          <x>6</x>
          <y>6</y>
          <width>768</width>
          <height>413</height>
         </rect>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>CipherText</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QHexView" name="cipherHexView">
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget" native="true">
      <layout class="QGridLayout" name="gridLayout">
       <item row="2" column="0">
        <widget class="QWidget" name="widget_2" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>PlainText Length</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit"/>
          </item>
         </layout>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QPushButton" name="pushButton">
         <property name="text">
          <string>Measure</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0" colspan="2">
        <widget class="QTableWidget" name="tableWidget">
         <row>
          <property name="text">
           <string>AES_NI</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>AES_Step</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>AES_Table</string>
          </property>
         </row>
         <column>
          <property name="text">
           <string>Cycle</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Cycle per Byte</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>32</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <widget class="QMenu" name="menuOpen">
     <property name="title">
      <string>Open</string>
     </property>
     <addaction name="actionOpenPlainText"/>
     <addaction name="actionOpenCipherText"/>
     <addaction name="actionOpenKey"/>
    </widget>
    <widget class="QMenu" name="menuSave">
     <property name="title">
      <string>Save</string>
     </property>
     <addaction name="separator"/>
     <addaction name="actionSavePlainText"/>
     <addaction name="actionSaveCipherText"/>
     <addaction name="actionSaveKey"/>
    </widget>
    <addaction name="menuOpen"/>
    <addaction name="menuSave"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <widget class="QMenu" name="menuEncrypt">
     <property name="title">
      <string>Encrypt</string>
     </property>
     <addaction name="actionAES_NI_Encrypt"/>
     <addaction name="actionAES_Step_Encrypt"/>
     <addaction name="actionAES_TableEncrypt"/>
    </widget>
    <widget class="QMenu" name="menuDecrypt">
     <property name="title">
      <string>Decrypt</string>
     </property>
     <addaction name="actionAES_NI_Decrypt"/>
     <addaction name="actionAES_Step_Decrypt"/>
     <addaction name="actionAES_TableDecrypt"/>
    </widget>
    <addaction name="menuEncrypt"/>
    <addaction name="menuDecrypt"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QDockWidget" name="dockWidget_4">
   <property name="windowTitle">
    <string>Key</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>8</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_5">
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QHexView" name="keyHexView">
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="actionAES_NI_Encrypt">
   <property name="text">
    <string>AES_NI</string>
   </property>
  </action>
  <action name="actionAES_NI_Decrypt">
   <property name="text">
    <string>AES_NI</string>
   </property>
  </action>
  <action name="actionOpenPlainText">
   <property name="text">
    <string>PlainText</string>
   </property>
  </action>
  <action name="actionOpenCipherText">
   <property name="text">
    <string>CipherText</string>
   </property>
  </action>
  <action name="actionOpenKey">
   <property name="text">
    <string>Key</string>
   </property>
  </action>
  <action name="actionSavePlainText">
   <property name="text">
    <string>PlainText</string>
   </property>
  </action>
  <action name="actionSaveCipherText">
   <property name="text">
    <string>CipherText</string>
   </property>
  </action>
  <action name="actionSaveKey">
   <property name="text">
    <string>Key</string>
   </property>
  </action>
  <action name="actionAES_Step_Encrypt">
   <property name="text">
    <string>AES_Step</string>
   </property>
  </action>
  <action name="actionAES_Step_Decrypt">
   <property name="text">
    <string>AES_Step</string>
   </property>
  </action>
  <action name="actionAES_TableEncrypt">
   <property name="text">
    <string>AES_Table</string>
   </property>
  </action>
  <action name="actionAES_TableDecrypt">
   <property name="text">
    <string>AES_Table</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QHexView</class>
   <extends>QFrame</extends>
   <header location="global">qhexview.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
